schema/template.schema.json

```
@@ -1046,7 +1046,7 @@
         },
         "type": {
           "$ref": "#/definitions/FieldType",
-          "description": "Type of the data field."
+          "description": "Type of the data filed."
```

```suggestion
          "description": "Type of the data field."
```

I guess it is a typo